text/0000-links-for-buildpacks.md

```diff
@@ -0,0 +1,56 @@
+# Meta
+[meta]: #meta
+- Name: Buildpack homepage
+- Start Date: 1-23-2020
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+This RFC suggest a mechanism where buildpacks can provide arbitrary links in their configuration: point to a homepage. These links are intended to contain documentation, point to github repos, point users to readmes, etc. but should not prescribe a format.
```

Comment:
I'm not sure I love the key `homepage` (can't decide to whether to host mine at Geocities or MySpace) but in the absence of something better, it's good enough.

Reply:
@nebhale lol, but i agree. That said, there seems to be plenty of prior art.

Reply:
Yeah me too. But old-skool is 😎sometimes.